main.tex

```diff
@@ -171,11 +182,9 @@
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  notify the Manager or the Owners Corporation of the danger or threat;
```

Comment:
Notice above (line 170) that this is the **immediate** task. The way this rule is worded, calling The Knight is more urgent than calling the fire brigade.